content/operate/kubernetes/reference/yaml/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: YAML examples
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: Example YAML files for deploying Redis Enterprise on Kubernetes with different configurations.
+hideListLinks: true
+linkTitle: YAML examples
+weight: 85
+---
+
+This section provides complete YAML examples for common Redis Enterprise for Kubernetes deployment scenarios. Each example includes the necessary configuration files and step-by-step instructions for editing and applying them.
+
+## How to use these examples
+
+### Download and customize
+
+1. Copy the YAML content from the examples below
+2. Save each YAML block to a separate file with a descriptive name
+3. Edit the configuration values to match your environment
+4. Apply the files in the correct order using `kubectl apply`
+
+### Configuration storage
+
+Redis Enterprise for Kubernetes stores configuration in several places:
+
+- Custom resources: Cluster and database specifications are stored as Kubernetes custom resources (REC, REDB, REAADB, RERC)
+- Secrets: Sensitive data like passwords and certificates are stored in Kubernetes secrets
+- ConfigMaps: Non-sensitive configuration data is stored in ConfigMaps
+- RBAC resources: Permissions are defined through Roles, ClusterRoles, and their bindings
+
+### Applying YAML files
+
+Apply YAML files using `kubectl apply`:
+
+```bash
+# Apply a single file
+kubectl apply -f my-config.yaml
+
+# Apply multiple files
+kubectl apply -f rbac/ -f cluster/ -f database/
+
+# Apply with validation
```

Comment:
This comment might incorrectly imply that the previous `apply` forms don't employ validation, and this one does, in addition to persistently applying the configuration in the manifest. The reality is kinda the opposite: all of these commands employ validation (both client- and server-side), and the last one only does client-side validation and doesn't actually apply the objects.

Hence, consider a comment in the likes of:
> \# Validate files without actually applying